docs/sources/setup-grafana/installation/redhat-rhel-fedora/index.md

```diff
@@ -75,6 +75,11 @@ If you wish to install beta versions of Grafana, substitute the repository URL f
    sudo dnf install grafana-enterprise
    ```
 
+1. When installing Grafana v11.2 or v11.3 on RHEL9 and RHEL9-based Linux distributions, remove the `grafana-server` init.d script:
+   ```bash
+   sudo rm /etc/rc.d/init.d/grafana-server
+   ```
```

Comment:
I'm not sure if this is the best way to present this information - I also would like to clarify with somebody somewhere that this is needed for installation on RHEL9 generally (although I'm pretty sure it is). I think this is a passable solution but maybe an admonition is more apt and there we can even provide some clarification as to why this is needed in these limited scenarios and not by-and-large? Just some thoughts

Reply:
@irenerl24 what do you think?

Reply:
I think an admonition would be a good idea so that we can explain why is it needed, but the solution you proposed now it's also suitable!